synfig-studio/src/gui/docks/dockmanager.cpp

```diff
@@ -535,6 +602,8 @@ Gtk::Widget* DockManager::read_widget(std::string &x)
 
 		if (!widget) return NULL;
 
+		check_and_fix_window_position(left, top, width, height);
```

Comment:
There is a strange problem here. The coordinates of the top left corner of the Synfig main window are 0, 217. At first I thought it was a Windows 10 issue or a GTK bug, but this example works fine and setting the coordinates to 0, 0 moves the window to the top left corner:

```
#include <gtkmm.h>

int main(int argc, char *argv[])
{
	Glib::RefPtr<Gtk::Application> app = Gtk::Application::create(argc, argv, "org.gtkmm.examples.base");

	Gtk::Window window;
	window.set_default_size(200, 200);
	window.move(0, 0);

	return app->run(window);
}
```

But for some reason the Synfig window works differently and the top corner goes out of the visible area.

Reply:
Sorry, I didn't understand.
I moved the synfig window, positioned the mouse cursor at top left corner, pressed Ctrl+Q to close it, and without moving the mouse pointer, I relaunched Synfig. The window opened in the exact position that mouse pointer marked.

I tested to close minimized window and it shows up at (0,0) on relaunch.

Reply:
This is how it looks with 0,0 coordinates.

P.S. This is possible GTK bug.

![diLSVY7xPW](https://user-images.githubusercontent.com/5604544/103974826-32488800-51a5-11eb-8e38-fd006cdc8f7f.gif)

Reply:
That wasn't like that when I tested on Windows o.O

Anyway, try to move the window after `present()` it.

Reply:
> Anyway, try to move the window after present() it.

Bug is reproducible when you try to `move()` after `present()`, lol )

And yes, the problem exists because this code is being called twice. I did not dig why this is happening, I will look later.

At the first time window positioned correctly and shows. If you call `move()` again it positions incorrectly. Reproduced only on multi-monitor configuration, if monitors have different DPI.

Reply:
@rodolforg please add temporary hack here:
```
		// FIXME(ice0): hack to fix incorrect positioning https://github.com/synfig/synfig/issues/1975
		if (App::main_window->is_visible()) App::main_window->hide();
		App::main_window->move(left, top);
		App::main_window->set_default_size(width, height);
```

Reply:
Done